xml/http_cookies.xml

```diff
@@ -127,10 +127,11 @@
   <fingerprint pattern="^NSC_(?:AAAC|CERT|DLGE|EPAC|TASS|TEMP|TMA[APS]|PERS)=.*">
     <description>Citrix NetScaler</description>
     <example>NSC_AAAC=xyz;</example>
-    <param pos="0" name="os.vendor" value="Citrix"/>
-    <param pos="0" name="os.family" value="NetScaler"/>
-    <param pos="0" name="os.device" value="Network Management Device"/>
-    <param pos="0" name="os.product" value="NetScaler"/>
+    <param pos="0" name="service.vendor" value="Citrix"/>
+    <param pos="0" name="service.family" value="NetScaler"/>
+    <param pos="0" name="service.device" value="Network Management Device"/>
+    <param pos="0" name="service.product" value="NetScaler"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:citrix:netscaler:-"/>
   </fingerprint>
```

Comment:
FYI @brudis-r7 - This fingerprint has changed quite a bit.

Reply:
This should probably assert os/hw like the rest of them